debian/changelog

```diff
@@ -31,14 +33,16 @@ cloud-init (22.2-145-gaa59209a-0ubuntu1~18.04.1) UNRELEASED; urgency=medium
     + tests: can run without azure-cli, tests expect inactive ansible
       (#1643)
     + typing: Type UrlResponse.contents (#1633) [Alberto Contreras]
-    + testing: fix references to `DEPRECATED.` (#1641) [Alberto Contreras]
+    + testing: fix references to `DEPRECATED.` (#1641) [Alberto
```

Comment:
I don't like that our current tooling with gbp is separating Alberto Contreras across two lines, can you manually bump his authorship name down to the next line in these few cases where it got separated.

Reply:
Done (and for the other branches as well).